docs/reference/edot-cloud-forwarder/aws.md

```diff
@@ -0,0 +1,182 @@
+::::
+
+In addition, you need to know the URL of the managed OTLP endpoint and the API key for authentication.
+
+:::{include} ../_snippets/serverless-endpoint-api.md
+:::
+
+## Configure the CloudFormation template
+
+{{edot-cf}} for AWS uses a CloudFormation template to deploy the EDOT Collector as a Lambda function.
+
+::::::{stepper}
+
+:::::{step} Download the CloudFormation template
+
+Download the `cloudformation.yaml` file from the [{{edot-cf}} for AWS GitHub repository](https://github.com/elastic/edot-cloud-forwarder-aws/blob/main/cloudformation.yaml).
+
+:::::
+
+:::::{step} Edit the settings in the template
+
+These are the required parameters you need to set in the CloudFormation template:
+
+| Parameter | Description |
+| --- | --- |
+| `region` | AWS region where the CloudFormation stack is to be deployed. |
+| `stack-name` | Name of the CloudFormation stack. For example, `vpc-edot-cf`. Do not use the same name for different sources. |
+| `EdotCloudForwarderSource` | Source of logs to be processed. Available options are `s3_logs` and `cloudwatch_logs`. |
+| `EdotCloudForwarderVersion` | Version of the EDOT Cloud Forwarder. Expected format is `0-1-2`, with hyphens replacing dots. Check available versions in the [GitHub repository](https://github.com/elastic/edot-cloud-forwarder-aws/releases). |
+| `ElasticOtlpEndpoint` | The Elastic OTLP endpoint URL used for data ingestion. |
+| `ElasticApiKey` | API key for authentication with Elastic. |
+
+Additionally, you need to set the following parameters based on the log source:
+
+::::{tab-set}
+
+:::{tab-item} S3
+
+If `EdotCloudForwarderSource` is `s3_logs`, set these options:
+
+| Parameter | Description |
+| --- | --- |
+| `EdotCloudForwarderS3Bucket` | The S3 bucket where logs are stored. This bucket will trigger the `edot-cloud-forwarder` Lambda function automatically. |
+| `EdotCloudForwardeS3LogsEncoding` | The encoding format for logs in the S3 bucket. Supported options:<br>• `vpc_flow_log` – VPC Flow Logs<br>• `elb_access_log` – Elastic Load Balancer (ELB) Access Logs<br>• `s3_access_log` – S3 Access Logs<br>• `json` – JSON-formatted logs |
+| `EdotCloudForwardeS3LogsJsonMode` | *(Required if `EdotCloudForwardeS3LogsEncoding` is `json`)*<br>Defines how JSON logs are structured:<br>• `body` *(default)* – Stores logs in the request body<br>• `body_with_inline_attributes` – Logs include inline attributes |
+
+:::
+
+:::{tab-item} CloudWatch
+
+If `EdotCloudForwarderSource` is `cloudwatch_logs`, set the following options:
+
+| Parameter | Description |
+| --- | --- |
+| `EdotCloudForwarderCloudWatchLogGroup` | Name of the CloudWatch Log Group where the subscription filter will be created. |
+
+:::
+::::
+:::::
+::::::
+
+### Optional settings
+
+These are optional parameters you can set in the CloudFormation template:
+
+| Parameter | Description |
+| --- | --- |
+| `EdotCloudForwarderLambdaName` | Set a custom name for the Lambda function. Default value is `edot-cloud-forwarder`. |
```

Comment:
Could we also add here a comment like in stack-name? 
```
Do not use the same name for different sources.
```
They should be careful not to try and create two lambda with same name for different use cases.